src/ui/README.md

```diff
@@ -4,14 +4,61 @@ The Aqueduct UI is implemented in Next.js. Once you've installed the Aqueduct
 package, you can start the UI by running `aqueduct ui`.
 
 The code here is organized into two directories -- a component library in
-`components/` that is published to npm, and an application server in `app/`
-that consumes `components/`.
+`common/` that is published to npm as a package called `@aqueducthq/common`, and an application server in `app/`
+that consumes `@aqueducthq/common`.
 
 If you're actively developing the UI, you will need to copy the `.env.local` file 
```

Comment:
@vsreekanti @agiron123  is the `.env.local` file still relevant? I don't think we have `~/.aqueduct/ui/app` folde anymore right? The directory just contains a few "compiled" UI files.

Reply:
@vsreekanti @cw75 just want to confirm that all we need in env.local is the SERVER_ADDRESS field, correct?
```
SERVER_ADDRESS=http://localhost:8080
```